running/ReplacingDocker/podman.sh

```diff
@@ -0,0 +1,311 @@
+    sudo podman top --latest seccomp
+    echo ""
+
+    read -p "--> sudo podman top --latest capeff"
+    sudo podman top --latest capeff
+    echo ""
+
+    read -p "--> clear"
+    clear
+}
+
+pods() {
+    sudo podman kill -a
+    sudo podman rm -a -f
+    clear
+
+    read -p "--> sudo podman pod list"
+    sudo podman pod list
+    echo ""
+
+    read -p "--> sudo podman pod --name podtest create"
+    sudo podman pod create --name podtest
+    echo ""
+
+    read -p "--> sudo podman pod create --pod podtest -d fedora sleep 600"
+    sudo podman create --pod podtest -d fedora sleep 600
+    echo ""
+
+    read -p "--> sudo podman pod create --pod podtest -d fedora sleep 600"
+    sudo podman create --pod podtest -d fedora sleep 600
+    echo ""
+
+    read -p "--> sudo podman ps"
+    sudo podman ps
+    echo ""
+
+    read -p "--> sudo podman pod start podtest"
+    sudo podman pod start podtest
+    echo ""
+
+    read -p "--> sudo podman ps"
+    sudo podman ps
+    echo ""
+
+    read -p "--> sudo podman pod stop podtest"
+    sudo podman pod kill --signal KILL podtest
+    echo ""
+
+    read -p "--> sudo podman ps"
+    sudo podman ps
+    echo ""
+
+    read -p "--> sudo podman pod rm --force podtest"
+    sudo podman pod rm --force podtest
+    echo ""
+    clear
+
+    read -p "--> sudo podman pod list"
+    sudo podman pod list
+    echo ""
+}
+
+build() {
+    echo ""
+    # Already built an image with buildah installed in it
+    # and made buildah the entrypoint
+
+    sudo mkdir -p /var/lib/mycontainer
+    mkdir -p $PWD/myvol
+    cat >$PWD/myvol/Dockerfile <<_EOF
+FROM alpine
+ENV foo=bar
+LABEL colour=blue
+_EOF
+
+    read -p "--> cat Dockerfile"
+    cat $PWD/myvol/Dockerfile
+    echo ""
+    echo ""
+    read -p "--> sudo podman run -v \$PWD/myvol:/myvol:Z -v /var/lib/mycontainer:/var/lib/containers:Z buildah-ctr --storage-driver vfs bud -t myimage --isolation chroot /myvol"
+    sudo podman run --net=host -v $PWD/myvol:/myvol:Z -v /var/lib/mycontainer:/var/lib/containers:Z buildah-ctr --storage-driver vfs bud -t myimage --isolation chroot /myvol
+    echo ""
+
+    read -p "--> sudo podman run -v /var/lib/mycontainer:/var/lib/containers:Z	buildah-ctr --storage-driver vfs images"
+    sudo podman run --net=host -v /var/lib/mycontainer:/var/lib/containers:Z buildah-ctr --storage-driver vfs images
+    echo ""
+
+    read -p "--> sudo podman run -v /var/lib/mycontainer:/var/lib/containers:Z	buildah-ctr --storage-driver vfs rmi --force --all"
+    sudo podman run --net=host -v /var/lib/mycontainer:/var/lib/containers:Z buildah-ctr --storage-driver vfs rmi -f --all
+    echo ""
+
+    read -p "--> cleanup"
+    echo "podman rm -a -f"
+    sudo podman rm -a -f
+    echo ""
+
+    read -p "--> clear"
+    clear
+}
+
+intro() {
+    read -p "Podman Demos!"
+    echo ""
+}
+
+setup
+
+buildah_image
+
+intro
+
+version
+
+build
+
+rootless
+
+userns
+
+forkexec
+
+top
+
+pods
+
+read -p "End of Demo"
```

Comment:
When the demo is done a number of images are left behind that cause issues if you rerun the demo.  You probably ought to rm/rmi -a -f them before exiting.

Reply:
They should not cause issues, I would rather not pull them down and create them every time,